launch_gz/setup.py

```diff
@@ -0,0 +1,48 @@
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'launch_gz'
+
+setup(
+    name=package_name,
+    version='0.246.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+    ],
+    package_data={'launch_gz': ['py.typed']},
+    install_requires=[
+        'setuptools',
+        'ament_index_python',
+        'launch',
```

Comment:
I think so because of `from launch_ros.substitutions import FindPackageShare
`

[7b80582](https://github.com/gazebosim/ros_gz/pull/548/commits/7b8058202f0f83e848c2985d1a4b72ef2c05e5cc)